ts_src/psetv2/updater.ts

```diff
@@ -128,6 +132,26 @@ export class Updater {
 
       if (input.reissuanceOpts)
         this.addInReissuance(inputIndex, input.reissuanceOpts);
+
+      if (input.explicitAsset) {
+        if (!input.explicitAssetProof)
+          throw new Error('missing explicitAssetProof');
```

Comment:
This check is already made by pset's sanity check method. We can avoid repeating it here.